### PR TITLE
Menu API changes

### PR DESCRIPTION
**Pull request by vidhireems**
1. Populated database with two collections - Menu and MenuItems
2. Created relevant Schema and Models
3. Developed GET APIs for retrieving Menu and Menu Items with possible exception   handling

### CODE REVIEW

File: interfaces/IMenuItemsModel.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Mongoose = require("mongoose");
+
+interface IMenuItemsModel extends Mongoose.Document {
+   
+    menuId: Number
+    restaurantId: Number
+    menu: [ {
+            category: string;
+            name: string;
+            price: number;
```
Comment (Harshithamlinganna) [inline interfaces/IMenuItemsModel.ts:10]: price is defined as a number and later in the model it is written as  price: String,

File: models/MenuModel.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    //constructor initilize the create schema and model
+    public constructor() {
+        this.createSchema();
+        this.createModel();
+    }
+
+    //function to create the schema for Menu
+    public createSchema(): void {
+        this.schema = new Mongoose.Schema(
```
Comment (Harshithamlinganna) [inline models/MenuModel.ts:26]: Can you add the testing o/p for this

File: interfaces/IMenuItemsModel.ts (hunk 1 above)
Comment (vidhireems) [reply interfaces/IMenuItemsModel.ts:10]: Updated. Thanks.

File: models/MenuModel.ts (hunk 2 above)
Comment (vidhireems) [reply models/MenuModel.ts:26]: I don't understand this request. Can you please elaborate on this further?

**Review by kapish-patel — APPROVED** [review]

**Pull request merged**